src/views/Home.vue

```diff
@@ -330,53 +328,59 @@ export default class Home extends Vue {
     this.dialog = true;
   }
 
-roundDate(d: Date) {
-  const date = moment(d);
-  date.millisecond(Math.floor(date.millisecond() / 1000) * 1000);
-  date.second(Math.floor(date.second() / 60) * 60);
-  date.minute(Math.round((date.minute() + 15) / 30) * 30 - 15);
-  return date.format("h:mma");
-}
+  roundDate(d: Date) {
+    const date = moment(d);
+    date.millisecond(Math.floor(date.millisecond() / 1000) * 1000);
+    date.second(Math.floor(date.second() / 60) * 60);
+    date.minute(Math.round((date.minute() + 15) / 30) * 30 - 15);
+    return date.format("h:mma");
+  }
 
   updateHistoricalAverages() {
-    let day = ""
+    let day = "";
     switch (this.dateTime.getDay()) {
```

Comment:
I know you didn't write this but `this.dateTime` would work better as a moment; this switch statement could turn into `day = this.dateTime.format("dddd");`. this could be assigned to a future PR

Reply:
yah, for sure I created an issue #41